Docs/Guide.md

```diff
@@ -44,9 +44,9 @@ Then, create a partition on the remaining space. By the end, your partition map
 
 ![Windows XP partition setup](Assets/Guide/QEMU_002.png)
 
-Now, you can install to the larger partition and go through the installation as normal. This should only take a few minutes unless you don't type your product key in correctly 37 times.
+Now, you can install to the larger partition and go through the installation as normal. This should only take a few minutes unless you type in your product key incorrectly 37 times.
```

Comment:
The only sentence where I'm not sure I got the idea of it correctly, since there's double negation.